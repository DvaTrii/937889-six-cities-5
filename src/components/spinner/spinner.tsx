import * as React from "react";
import './spinner.css';

export const Spinner = () => {
  return (
    <div className="lds-ring">
      <div></div>
      <div></div>
      <div></div>
      <div></div>
    </div>
  );
};
