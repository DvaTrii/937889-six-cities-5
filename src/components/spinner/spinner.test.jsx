import React from "react";
import renderer from "react-test-renderer";
import {Spinner} from "./spinner";

it(`Should Spinner render properly`, () => {

  const tree = renderer
    .create(
        <Spinner />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
