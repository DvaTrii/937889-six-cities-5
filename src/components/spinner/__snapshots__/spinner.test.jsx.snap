// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Spinner render properly 1`] = `
<div
  className="lds-ring"
>
  <div />
  <div />
  <div />
  <div />
</div>
`;
